misc/codegen/test/test_schemaloader.py

```diff
@@ -774,6 +775,63 @@ class _:
     }
 
 
+def test_annotate_fields():
+    @load
```

Comment:
Adding additional annotation to a class doesn't seem to work yet. I tried:
```python
@annotate(LetExpr)
@rust.doc_test_signature("(maybe_some: Option<String>) -> ()")
class _:
```
but the tests still ends up with a `() -> ()` signature.

Reply:
yep, I initially forgot that, fixed now :slightly_smiling_face: